include/osquery/system.h

```diff
@@ -32,6 +32,18 @@ namespace osquery {
  */
 extern volatile std::sig_atomic_t kExitCode;
 
+#ifdef WIN32
+/// Unfortunately, pid_t is not defined in Windows, however, DWORD is the
+/// most appropriate alternative since process ID on Windows are stored in
+/// a DWORD.
+using pid_t = unsigned long;
```

Comment:
NB: This is to avoid including `Windows.h` within our public API headers. //cc @PoppySeedPlehzr please don't be mad. The net negative effect is an additional two static casts for the `HMODULE` type.